src/ResourceManager/Insights/Commands.Insights/LogProfiles/AddAzureRmLogProfileCommand.cs

```diff
@@ -56,48 +56,57 @@ public class AddAzureRmLogProfileCommand : ManagementCmdletBase
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The retention in days")]
         [ValidateNotNullOrEmpty]
-        public int? RetentionInDays { get; set; }
+        [Alias("RetentionInDays")]
+        public int? RetentionInDay { get; set; }
```

Comment:
@gucalder for parameters with units (_e.g.,_ `InDays`, `InMinutes`, etc.), you don't have to worry about making them singluar